act/retrievals/sirs_calculations.py

```diff
@@ -0,0 +1,187 @@
+
+""" Calculating SIRS Radiation Estimates """
+
+import numpy as np
+from astral import sun
+from scipy.constants import Stefan_Boltzmann
+from datetime import datetime, timedelta
+import astral
+
+def numpy_to_datetime(date_time64_val):
```

Comment:
This function is already include in the datetime_utils.py moduledatetime64_to_datetime.  We should remove from here and use the existing function.

Reply:
Okay I will change this!